components/Hero.jsx

```diff
@@ -0,0 +1,32 @@
+import Image from 'next/image';
+import React from 'react';
+import IgImg1 from '../public/ig-img-1.png';
+import Galleryimg from './Galleryimg'
+
+const Hero = ({heading,message}) => {
+  return (
+    <div>
+        {/* overlay */}
+    <div />
+       <div className='flex items-center justify-between h-screen mb-[10rem] bg-fixed bg-center bg-cover custom-img'>
+            <div className='container ml-36 gap-60 sm:flex md:flex md:container'>
+                <div className='flex-row mt-64 z-40 w-96 '>
+                    <h2 className='text-5xl'>{heading}</h2>
+                    <p className='mt-5 mb-10 '>{message}</p>
+                    <div className=' flex-col'>
+                        <button className='bg-cyan-800 gap-2 rounded-lg text-white mr-2 py-3 px-5'>Services</button>
+                        <button className='bg-slate-300 rounded-lg  py-3 px-5' >About the company</button>
+                    </div>
+                </div>
+                <div className='w-[40%] mb-10 flex-row mt-36 sm:w-[40%] '>
+                    <Galleryimg socialImg={IgImg1} />
+                </div>
```

Comment:
- If you decide to set your own values to different tailwind classes, try to not use arbitrary values, i.e: `class-[arbitrary value]`. Configure them in **`tailwind.config.js`** because it is considered the best practice when it comes to passing styles directly in `Html` files. 

- Also, there is no need to set your own styles that already come with tailwind classes. Check out the hint below to get a clear understanding of what I am saying. 

**Hint** : 
- Since `w-[40%]` is the same as `w-2/5 ` in tailwind, use such built-in tailwind class.